packages/material-ui/src/Box/Box.spec.tsx

```diff
@@ -29,6 +29,20 @@ function ResponsiveTest() {
   </Box>;
 }
 
+function GapTest() {
+  <Box
+    sx={{
+      width: '100%',
+      display: 'flex',
+      alignItems: 'center',
+      flex: '1 0',
+      gap: (theme) => theme.spacing(2),
```

Comment:
> the typings are working as expected without any changes. Should we still apply:

The author has updated his reproduction (testing the proposed workaround)

```suggestion
      gridGap: (theme) => theme.spacing(2),
```

Reply:
> Should we still apply

Your proposed alternative fix looks good.

Reply:
Ah sorry, my understanding was that we are going to move away from `gridGap` and should not support it, but I see that that is not the case. Let me update the test then